packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/src/testRunOperations/testRunOperationsClient.ts

```diff
@@ -150,32 +150,48 @@ export class TestRunOperationsClient {
     testRunId: string,
     fileName: string,
     options: GetTestRunFileOptionalParams = { requestOptions: {} },
-  ): Promise<FileInfo> {
+  ): Promise<{
+    url?: string;
+    fileType?: FileType;
+    expireDateTime?: Date;
+    validationStatus?: FileStatus;
+    validationFailureDetails?: string;
+  }> {
     return getTestRunFile(this._client, testRunId, fileName, options);
   }
 
   /** List the dimension values for the given metric dimension name. */
   listMetricDimensionValues(
     testRunId: string,
     name: string,
+    metricname: string,
     metricNamespace: string,
+    timespan: string,
     options: ListMetricDimensionValuesOptionalParams = { requestOptions: {} },
-  ): PagedAsyncIterableIterator<DimensionValueList> {
+  ): Promise<DimensionValueList> {
```

Comment:
Is changing from pagable to non-pagable a breaking? Also why would they have this change, it should be common to see changing from non-pagable to pagable, but should be uncommon in the reverse case?

Reply:
@weidongxu-microsoft Not sure you have more context. In the latest loadtesting spec,
- there is no LRO now
- listMetricDimensionValues is changed from ResourceList to Operation but becoming non-paging

Reply:
The LRO in Java is handwritten
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/loadtesting/azure-developer-loadtesting/src/main/java/com/azure/developer/loadtesting/LoadTestRunAsyncClient.java#L127-L146

(from what I remember, the status etc. does not match Azure guideline and SDK core lib handling of LRO -- that is the api-version, this version maybe different)

Reply:
The LRO change is expected and the paging change is waiting for the service team's reply, but I think we don't need to block this pr. Let me offline confirm this.